addons/nuxeo-retention/nuxeo-retention-core/src/main/java/org/nuxeo/retention/operations/UnholdDocument.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Guillaume RENARD
+ */
+package org.nuxeo.retention.operations;
+
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.collectors.DocumentModelCollector;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.DocumentRef;
+
+/**
+ * @since 11.1
+ */
+@Operation(id = UnholdDocument.ID, category = Constants.CAT_DOCUMENT, label = "Unhold", description = "Remove a legal hold on the input document. Returns back the unhold document.")
+public class UnholdDocument {
+
+    public static final String ID = "Document.Unhold";
+
+    @Context
+    protected CoreSession session;
+
+    @OperationMethod(collector = DocumentModelCollector.class)
+    public DocumentModel run(DocumentRef doc) {
+        session.setLegalHold(doc, false, null);
+        return session.getDocument(doc);
+    }
+
+    @OperationMethod(collector = DocumentModelCollector.class)
+    public DocumentModel run(DocumentModel doc) {
+        return run(doc.getRef());
+    }
+
```

Comment:
I am wondering about these different operations, if we need to add the management of the save or not. I had the remark on my operations for Glacier PRs, it's why I ask :):
`@Param(name = "save", required = false, values = "false")`

Reply:
Let's keep the same API than on `10.10` for now. If we identify use cases where it will needed, we'll change on both `10.10` and `master` with a dedicated JIRA ticket